### ISSUE TITLE
Supplier Fluid Pipes and GregTech machines

### ISSUE THREAD

**Issue by JasonMcRay** [RS485/LogisticsPipes#791]
Supplier Fluid Pipes arent able to keep GT machines filled with certain amount of fluid.

How to reproduce:
1) Put a Fluid Provider Pipe next to Tank with Water
2) Put a Fluid Supplier Pipe next to for example GregTech Basic Cutting Machine
3) Set the supplier pipe to supply 500mb of water and Min. mode to None.
4) Provider Pipe will extract 500mb of Water from the tank and will put it into Cutting Machine
5) But it will keep requesting water until the internal tank of Cutting Machine will be full and the water is stuck inside the Supplier Pipe.

Setting this for for example classic BC Tank works fine. It keeps 500mb of Water in it until its emptied, then it will refill it back to 500mb.

**Comment by hron84** [comment]
Could you please add affected versions to your report, especially Forge, LP and GT version. I guess it's about InfiTech modpack, but it would be make easier the investigation.

**Comment by JasonMcRay** [comment]
Sure
Forge: 1490
LP: 0.9.23
GT: 5.09.01 (also 5.08.28)

**Comment by JasonMcRay** [comment]
I dont know if there is anything else needed from me. It is still happening with GT 5.09.22 and LP 0.9.3.126 and forge 1614.

**Comment by hron84** [comment]
@theZorro266 I think we waiting for someone else to confirm or reproduce this problem. Also developers are busy with a lot of things, so please be patient.

**Comment by bziemons** [comment]
This is not a bug, because there is no support for any Logistics Pipes and GregTech interaction.
"Needs Comments" in this case just means that someone can look at it and see, if it's on our side.

In general: I think we rely on the Forge API for fluid tanks. So if we ask how much is in the tank, the tank (in this case the GT machine) gives us the information. This just does not look like it's on our side.

**Comment by Blood-Asp** [comment]
Right, default Forge fluid support will not work for GT in this case. The reason is, that GT has two internal tanks. One for input and one for output. Only the output tank is visible with the forge fluid api, while the input tank would need special GT code. 
Nessary example code for that would be this: https://github.com/Blood-Asp/GT5-Unofficial/blob/experimental/src/main/java/gregtech/common/items/behaviors/Behaviour_Plunger_Fluid.java#L47-L53

**Comment by hron84** [comment]
@theZorro266 I didn't wanted to reply to you, I meant Jason, but perhaps I misclicked. Really sorry. 

@Blood-Asp This fact keeps thinking me GT is hard to interact with. Refusing implementing standard APIs is not how a mod should be developed.

**Comment by Blood-Asp** [comment]
What refusing standart APIs? The Forge Fluid API is used as far as possible, but it does not support multiple tanks in one block. 

There is no mod able to do that. Telling off GT here is inappropriate.

**Comment by hron84** [comment]
@Blood-Asp It's not precise, as far as I know, Forge Fluid API contains a ForgeDirection suppport so you can definitely query more than one tank via the API, based on the direction. This can be used to provide different tank content depending on the direction. Fluid API lacks only one support: providing more fluids from a single tank. But it is definitely not that case.
